src/app/router/connectorRoutes.tsx

```diff
@@ -21,95 +21,98 @@ import ConnectRaspiBlitz from "@screens/connectors/ConnectRaspiBlitz";
 import ConnectUmbrel from "@screens/connectors/ConnectUmbrel";
 import ConnectStart9 from "@screens/connectors/ConnectStart9";
 import ConnectMyNode from "@screens/connectors/ConnectMyNode";
+import i18n from "~/i18n/i18nConfig";
 
 const galoyPaths: { [key: string]: keyof typeof galoyUrls } = {
   bitcoinBeach: "galoy-bitcoin-beach",
   bitcoinJungle: "galoy-bitcoin-jungle",
 };
 
+const namespaceI18n = { ns: "welcome" };
+
 export default [
   {
     path: "create-wallet",
     element: <NewWallet />,
-    title: "Alby Wallet",
-    description: "Create or login to your Alby account",
+    title: i18n.t("choose_connector.alby", namespaceI18n),
+    description: i18n.t("choose_connector.alby_desc", namespaceI18n),
```

Comment:
nitpick: as the key is called `description` I would not use a short version (`desc`). 
I often find the short version more confusing and while developing one then has to always check and think how to write thing. 
Thus generally I like the normal english word better. - description is also not thaaaat much longer than desc. 

maybe `choose_connector.alby.title` and `choose_connector.alby.description` would also be more descriptive as key?` (same in the other places? - but I don't know what's a good practice there)

Reply:
Good point, I debated since the keys were getting long but you're right, it doesn't save that much space.